components/Nav/index.js

```diff
@@ -1,26 +1,40 @@
 import { useRouter } from "next/router";
 import { StyledNav, StyledUl, StyledLi, StyledLink, StyledIcon } from "./style";
 import { MdAddLocationAlt } from "react-icons/md";
-import { PiWavesBold, PiWavesFill } from "react-icons/pi";
+import {
+  PiWavesBold,
+  PiUserCircleBold,
+  PiUserCircleFill,
+  PiSignOutBold,
+} from "react-icons/pi";
 
 export default function Navigation() {
   const router = useRouter();
+
   const linksData = [
     {
       id: 1,
       title: "YOUR HIDDEN SPOTS",
-      href: "/",
+      href: "/yoursurfspots",
       icon: PiWavesBold,
       ariaLabel: "Navigate to your hidden spots",
     },
     {
       id: 2,
-      title: "ADD A NEW SPOT",
-      href: "/AddANewSpot",
+      title: "CREATE A NEW SPOT",
+      href: "/createspot",
       icon: MdAddLocationAlt,
       ariaLabel: "Navigate to add a new spot page",
     },
+    {
+      id: 3,
+      title: "PROFILE",
+      href: "/profile",
+      icon: PiUserCircleBold,
+      ariaLabel: "Navigate to your profile",
+    },
   ];
+
```

Comment:
Very nice solution for creating the navigation! 👍

Reply:
THX